sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.MsTest.cs

```diff
@@ -43,20 +43,6 @@ public void TestMethod5()
             x.Should().Be(42);
         }
 
-        [TestMethod]
```

Comment:
this was moved in `TestMethodShouldContainAssertion_NUnit_FluentAssertionsLegacy`